redash/tasks/schedule.py

```diff
@@ -62,6 +63,11 @@ def schedule(kwargs):
 def periodic_job_definitions():
     jobs = [
         {"func": refresh_queries, "interval": 30, "result_ttl": 600},
+        {
+            "func": remove_ghost_locks,
+            "interval": timedelta(minutes=1),
+            "result_ttl": 600,
```

Comment:
just curious: does it mean this will run every minute for a maximum time of 10 minutes?

Reply:
Yep, after `result_ttl` (which defaults to `interval * 2` in Redash) seconds, the job and any results will be cleaned up.